CHANGELOG.md

```diff
@@ -1,4 +1,91 @@
+   - **aviatrix_distributed_firewalling_config**
+6. Implemented a new resource to support Distributed Firewalling Intra VPC:
+   - **aviatrix_distributed_firewalling_intra_vpc**
+
+#### Security
+1. Implemented a new resource to support FQDN Global Configs:
+   - **aviatrix_fqdn_global_config**
+
+### Enhancements:
+1. Updated ``transit_gateway_name`` to optional in **aviatrix_segmentation_network_domain_association**
+2. Removed validation for controller IP address during import in **aviatrix_controller_security_group_management_config**
+3. Updated **aviatrix_app_domain** to **aviatrix_smart_group**
+4. Updated **aviatrix_microseg_policy_list** to **aviatrix_distributed_firewalling_policy_list**
+5. Added API key for server and client authentication
+
+### Bug Fixes:
+1. Fixed issue where configuring large number of policies errors out in **aviatrix_gateway_dnat**
+2. Fixed issue where delta shows on ``region`` when same ``region`` is provided in different formats
+3. Fixed issue where ``rx_queue_size`` does not apply on HA in **aviatrix_gateway**, **aviatrix_spoke_gateway** and **aviatrix_transit_gateway**
+4. Fixed issue where route table list is missing in state file while importing **aviatrix_aws_peering**
+5. Fixed issue where HA creation does not use the right CMK for volume encryption when CMK is provided in **aviatrix_spoke_gateway** and **aviatrix_transit_gateway**
+
+### Deprecations
+1. The following resources are removed:
+   - **aviatrix_arm_peer**
+   - **aviatrix_aws_tgw_security_domain**
+   - **aviatrix_aws_tgw_security_domain_connection**
+   - **aviatrix_edge_caag**
+   - **aviatrix_segmentation_security_domain**
+   - **aviatrix_segmentation_security_domain_association**
+   - **aviatrix_segmentation_security_domain_connection_policy**
+   - **aviatrix_spoke_vpc**
+   - **aviatrix_transit_vpc**
+2. Removed support for the following attribute from the resources **aviatrix_gateway_snat** and **aviatrix_gateway_dnat**:
+   - ``sync_to_ha``
+3. Removed support for the following attribute from the resource **aviatrix_gateway** and the data source **aviatrix_gateway**:
+   - ``tag_list``
+4. Removed support for the following attribute from the resource **aviatrix_transit_gateway** and the data source **aviatrix_transit_gateway**:
+   - ``tag_list``
+5. Removed support for the following attributes from the resource **aviatrix_spoke_gateway**:
+   - ``manage_transit_gateway_attachment``
+   - ``transit_gw``
+   - ``tag_list``
+6. Removed support for the following attributes from the data source **aviatrix_spoke_gateway**:
+   - ``transit_gw``
+   - ``tag_list``
+7. Removed support for the following attributes from the resource **aviatrix_firenet**:
+   - ``manage_firewall_instance_association``
+   - ``firewall_instance_association``
+8. Removed support for the following attributes from the data source **aviatrix_firenet**:
+   - ``firewall_instance_association``
```

Comment:
You can do the same as the example above for ``firewall_instance_association`` here.

5. Removed support for managing in-line firewall instance associations by removing the following attributes from the resource **aviatrix_firenet** and its respective data source:
   - ``manage_firewall_instance_association`` (from the resource)
   - ``firewall_instance_association`` (from both)